subcommand/control-plane/command.go

```diff
@@ -67,6 +67,7 @@ type Command struct {
 
 const (
 	dataplaneConfigFileName = "consul-dataplane.json"
+	caCertFileName          = "mesh-task-consul-ca-cert.pem"
```

Comment:
nit: Naming suggestion

```suggestion
	caCertFileName          = "consul-grpc-ca-cert.pem"
```